server/channels/api4/ip_filtering.go

```diff
@@ -81,7 +81,10 @@ func applyIPFilters(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	auditRec.Success()
 
-	go c.App.SendIPFiltersChangedEmail(c.AppContext, c.AppContext.Session().UserId)
+	if err := c.App.SendIPFiltersChangedEmail(c.AppContext, c.AppContext.Session().UserId); err != nil {
+		c.Err = model.NewAppError("SendIPFiltersChangedEmail", "api.context.ip_filtering.send_email.app_error", nil, "", http.StatusInternalServerError).Wrap(err)
+		return
+	}
```

Comment:
The previous code was running in a separate go routine. Can you please add the back?

Reply:
It's fine to log the error instead of returning it. The response has already been send when `SendIPFiltersChangedEmail` returns.